lib/pbench/server/api/resources/query_apis/iteration_samples.py

```diff
@@ -163,23 +151,23 @@ def assemble(self, json_data: JSON, context: CONTEXT) -> JSON:
 
     def postprocess(self, es_json: JSON, context: CONTEXT) -> JSON:
         """
-        Returns all iteration samples as json containing the aggregation data.
+        Returns all iteration samples as JSON containing the aggregation data.
         The 2 top level keys are:
-        "hits": Continaing information about total number of hits and
-                list of all the iterations samples json across all the scrolled pages
+        "hits": Containing information about total number of hits and
+                list of all the iterations samples as JSON returned from the ES query.
```

Comment:
I would omit "returned from the ES query", but if you want to keep it, I would replace "query" with "query(es)" or "queries".